### PR TITLE
I changed some selectors

### PR DESCRIPTION
**Pull request by juanjo44**

### CODE REVIEW

File: cypress/page/menu-content.page.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ class MenuContentPage {
     private menuContentPageURL: string
 
     constructor() {
-        this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+        this.tShirtMenu = ".sf-menu > :nth-child(3) > a";
```
Comment (holgiosalos) [inline cypress/page/menu-content.page.ts:6]: the key here, is not depend on the position of any item, you can preserve the beginning of the old one `#block_top_menu > ul > li` but then look for an attribute of the `a` tag that can be used to indicate that we are selecting the t-shirts option

File: cypress/page/products-list.page.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
   private checkout: string;
 
   constructor (){
-    this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkout = "[style*='display: block;'] .button-container > a";
+    this.addToCartButton = ".ajax_add_to_cart_button > span";
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts:6]: you can use an attribute here to indicate we are clicking on the `Add to cart` button

File: cypress/page/products-list.page.ts [hunk 3]
```diff
@@ -3,8 +3,8 @@ class ProductsListPage {
...
+    this.checkout = ".button-container > .button-medium > span";
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts:7]: here you can use the text of the button and then a contains in the method that this selector is used, or, you can use an attribute to indicate you are referring to the Proceed to checkout button.

**Review by holgiosalos — COMMENTED** [review]
The following page objects also can be improved:

1. AddressStepPage
2. ShippingStepPage 
3. ShoppingCartPage

File: cypress/page/products-list.page.ts [hunk 4]
```diff
@@ -3,16 +3,16 @@ class ProductsListPage {
   private checkout: string;
 
   constructor (){
-    this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.checkout = "[style*='display: block;'] .button-container > a";
+    this.addToCartButton = "a[title='Add to cart'";
```
Comment (manuelq12) [inline cypress/page/products-list.page.ts:6]: I think a `]` is missing for the addToCartButton

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/page/products-list.page.ts (hunk 4 above)
Comment (holgiosalos) [reply cypress/page/products-list.page.ts:6]: yep

**Review by holgiosalos — COMMENTED** [review]
apply the change suggested by @manuelq12 to approve it

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**